### ISSUE TITLE
Error will always be nil

### ISSUE THREAD

**Issue by olamiko** [yhabib/csi#3]
In the function `readInput`, error will always be nil - https://github.com/yhabib/csi/blob/658526d5315ec219fcd95c0c04bdc01ddd80fc35/prework/Go/assignment/xkdc.go#L67. What's the point of checking for error at [line](https://github.com/yhabib/csi/blob/658526d5315ec219fcd95c0c04bdc01ddd80fc35/prework/Go/assignment/xkdc.go#L41) then

**Comment by yhabib** [comment]
Good point, I think I first approached differently and then forgot to clean up the code.
